cmd/gen/makefile/template/template.go

```diff
@@ -0,0 +1,67 @@
+package template
+
+//TemplateConfig inherits all placeholder which should be overwritten.
+type TemplateConfig struct {
+	Application string
+}
+
```

Comment:
Because of consistency I would be happier if we followed the pattern like [here](https://github.com/giantswarm/devctl/blob/master/pkg/gen/input/ami/internal/file/ami.go).

Even though I understand it may feel like an overkill for this particular case.

If you'd like to do so then we need to make this file regenerable [here](https://github.com/giantswarm/devctl/blob/c5630628e98c5fb33582f1511a8f02332c90b50d/pkg/gen/gen.go#L44-L52).

But I won't block on that because I think it's already an improvement.